config/deploy/server_settings.rb

```diff
@@ -1,31 +1,9 @@
 # Note: this file is required in config/deploy/<environment>.rb
 
 puts "ENV['APP_ENV']  = #{ENV['APP_ENV']}"
-puts "ENV['SDR_APP']  = #{ENV['SDR_APP']}"
-puts "ENV['SDR_USER'] = #{ENV['SDR_USER']}"
-puts "ENV['SDR_HOST'] = #{ENV['SDR_HOST']}"
-puts "ENV['SDR_URL']  = #{ENV['SDR_URL']}"
 puts
 
-set :default_env, {
-    'APP_ENV'  => ENV['APP_ENV'],
-    'SDR_APP'  => ENV['SDR_APP'],
-    'SDR_USER' => ENV['SDR_USER'],
-    'SDR_HOST' => ENV['SDR_HOST'],
-    'SDR_URL'  => ENV['SDR_URL'],
-}
-
-server ENV['SDR_HOST'], user: ENV['SDR_USER'], roles: %w{app}
-Capistrano::OneTimeKey.generate_one_time_key!
-ssh_opts = fetch(:ssh_options)
-ssh_opts[:forward_agent] = true
-ssh_opts[:verbose] = false
-# The :ssh_options are set by
-# capistrano-one_time_key/blob/master/lib/capistrano/tasks/one_time_key.rake
```

Comment:
@cbeer and @eefahy might be able to help with that; I forget why we had to add them in, but they work and we forgot all about it, sayin "if it ain't broke, don't fix it".  If we take them out, then we should run a cap deploy to the `stage` platform to test it out AND, if the prod systems are in a different (more secure) zone, we also need a way to check it for prod too.  (My memory is really vague on why we did this a few years ago now, but it might have something to do with PROD secure zone stuff.)